### PR TITLE
Admin dashboard template

### PR DESCRIPTION
**Pull request by alexgul1**
Added admin dashboard template (page with list of apps, app page, social network page)
added test stubs and snapshots for created pages

### CODE REVIEW

File: src/components/AppInfo/AppInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+const AppInfo = () => {
+  const {url} = useRouteMatch();
+
+  return (
+    <Container>
+      <div className={styles.appPreview}>
+        <img src={appIcon} className={styles.appIcon} alt="icon"/>
+        <h2>App name</h2>{/*TODO from backend*/}
+      </div>
```
Comment (Ikilonchic) [inline src/components/AppInfo/AppInfo.jsx:32]: Add vertical centering.

File: src/components/SocialInfo/SocialInfo.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import styles from "./SocialInfo.module.scss";
+import socialIcon from "../../assets/google-avatar.png";
+
+const SocialInfo = () => {
+  return (
+    <Container>
+      <div className={styles.socialPreview}>
+        <img src={socialIcon} className={styles.socialIcon} alt="icon"/>
+        <h2>Social network name</h2>{/*TODO get from back*/}
+      </div>
```
Comment (Ikilonchic) [inline src/components/SocialInfo/SocialInfo.jsx:14]: Add vertical centering.

File: src/components/SocialInfo/SocialInfo.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          </FormGroup>
+          <FormGroup>
+            <FormLabel>Allowed Mobile Client IDs</FormLabel>
+            <FormControl as={'textarea'} rows={3}/>
+            <FormText className={'text-muted'}>You can specify multiple valid client IDs by comma-separating
+              them</FormText>
+          </FormGroup>
+
+          <Button variant={"info"}>Update</Button>
+        </Col>
```
Comment (Ikilonchic) [inline src/components/SocialInfo/SocialInfo.jsx]: This column shrink on mobile.

File: src/components/AppList/AppList.jsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  return (
+    <Container>
+      <Row className='mb-5'>
+        <Col>
+          <h2>Applications</h2>
+        </Col>
+        <Col xs="auto">
+          <CreateAppModal />
+        </Col>
+      </Row>
```
Comment (Ikilonchic) [inline src/components/AppList/AppList.jsx]: Add vertical centering here, too :)

**Review by Ikilonchic — CHANGES_REQUESTED** [review]

**Review by matroskin062 — COMMENTED** [review]
Agree with comments of @Ikilonchic

**Comment by alexgul1** [comment]
update 8cb51face7df5fd1a14f9d8dd85a752df7d7006e

**Review by matroskin062 — APPROVED** [review]
Looks good 😎

**Review by Ikilonchic — APPROVED** [review]

**Pull request merged**